lib/codegen/hotReload.js

```diff
@@ -19,7 +19,7 @@ if (module.hot) {
   api.install(require('vue'))
   if (api.compatible) {
     module.hot.accept()
-    if (!module.hot.data) {
+    if (!api.isRecorded('${id}')) {
```

Comment:
could just be `api.isRecorded(id)`

Reply:
Sorry, couldn't get it. Did you mean to remove the `${}`? I don't think that could work.

Reply:
`id` is already a string isn't it?

Reply:
Yes, but we are in a template string. You can see the codes below this line also does the same thing. Let me know if I miss anything.